### DISCUSSION TITLE
API Pariwisata Indonesia

### DISCUSSION THREAD

**Discussion by putra4648** [farizdotid/DAFTAR-API-LOKAL-INDONESIA#324] [General]
mas kira kira ada gk API buat Pariwisata Indonesia?

**Comment by farizdotid** [comment]
@pradana4648 barusan saya searching belum nemu euy mas.

**Comment by putra4648** [comment]
waduuu susah juga ya ternyata

**Comment by farizdotid** [comment]
gpp mas jangan di close, siapa tau temen disini ada yang kasih info hehe

**Comment by putra4648** [comment]
ok ok

**Comment by wanrabbae** [comment]
@pradana4648 cuma nemu data JSON
https://github.com/ypxio/noesantara/blob/master/res/data/pariwisata.json

**Comment by putra4648** [comment]
@wanrabbae mungkin suatu saat akan saya bikin API sederhananya di lain waktu

**Comment by void-vim** [comment]
Info cari juga 
Klo ada kabarin euy

**Comment by sosifast** [comment]
saya sedang membuat, segere release
